askama/src/filters/alloc.rs

```diff
@@ -820,6 +820,31 @@ fn flush_title(dest: &mut (impl fmt::Write + ?Sized), s: &str) -> fmt::Result {
     Ok(())
 }
 
+/// Return a title cased version of the value. Alias for the [`|title`](title) filter.
+///
+/// ```
+/// # #[cfg(feature = "code-in-doc")] {
```

Comment:
I think we never run these doc tests because of this `cfg`.

Reply:
Hm, you're right. I (hope I) fixed that in <https://github.com/askama-rs/askama/pull/416>. I cherry-picked this PR over there.